src/Service/CaseManager.php

```diff
@@ -99,4 +103,47 @@ public function newCase(CaseEntity $caseEntity, Board $board): CaseEntity
 
         return $caseEntity;
     }
+
+    public function updateDeadlineBooleans($isDryRun)
+    {
+        // We do this in batches as there might be a large number of cases
+        // @see https://www.doctrine-project.org/projects/doctrine-orm/en/2.9/reference/batch-processing.html#iterating-results
+        $batchSize = 20;
+        $i = 1;
+        $query = $this->caseRepository->createQueryBuilder('c')->getQuery();
+
+        $today = new DateTime('today');
+
+        $this->logger->info('Today: '.$today->format('d/m/Y'));
+
+        foreach ($query->toIterable() as $case) {
+            assert($case instanceof CaseEntity);
```

Comment:
Is this necessary? Isn't `$case` always of type `CaseEntity`?

Reply:
It is, but the code analyzer may not be able to see this – and this may be to help the tool.

Reply:
Ah ok.

Reply:
Correct, this will always be of type `CaseEntity`. This was purely for helping the code analyzer and myself.